reference/docs-conceptual/PowerShell-Core-Support.md

```diff
@@ -0,0 +1,53 @@
+# PowerShell Core Support Lifecycle
+
+PowerShell Core is distinct from Windows PowerShell in that it is a separately shipping set of tools and components, and is therefore not included in the Windows 10 or Windows Server licensing agreements.
+
+However, it is also supported under traditional Microsoft support agreements, including [Premier][Premier support] and [Microsoft Enterprise Agreements][Enterprise agreement].
+You can also pay for [assisted support][] for PowerShell Core by filing a support request for your problem.
+
+We also offer [community support][] on GitHub where you can file an issue, bug, or feature request,
+but we offer no guarantee there that your issue will be addressed or resolved in a timely manner.
+If you have a problem that requires immediate attention,
+you should use the traditional, paid support options offered above.
+
+[Premier support]: https://www.microsoft.com/en-us/microsoftservices/support.aspx
+[Enterprise agreement]: https://www.microsoft.com/en-us/licensing/licensing-programs/enterprise.aspx
+[Community support]: https://github.com/powershell/powershell/issues
+[assisted support]: https://support.microsoft.com/en-us/assistedsupportproducts
+
+## Lifecycle of PowerShell Core
+
+PowerShell Core is adopting the [Microsoft Modern Lifecycle Policy][], a new support lifecycle intended to keep customers up-to-date with the latest versions.
+
+The 6.x branch of PowerShell Core will be updated approximately once every six months (e.g. 6.0, 6.1, 6.2, etc.)
+**After each new minor version release,**
+**you must update within six months in order to continue receiving support.**
+
+For example, if PowerShell Core 6.1 is released on July 1st, 2018,
+you would be expected to update off of PowerShell Core 6.0 by January 1st, 2019 in order to stay supported.
+
+![PowerShell Core branch lifecycle][]
+
+Eventually, we expect PowerShell Core to fork into something resembling a "long-term servicing" branch where we would require only servicing and security updates to stay in support.
+
+[Microsoft Modern Lifecycle Policy]: https://support.microsoft.com/en-us/help/447912/announcing-microsoft-modern-lifecycle-policy
+[PowerShell Core branch lifecycle]: TODO
+
+## Compatibility with Windows PowerShell modules
+
+Support for PowerShell Core does not extend to other product modules unless they explicitly support PowerShell Core.
+For example, using the `ActiveDirectory` that ships `as part of Windows Server is an unsupporgted scenario.
```

Comment:
typo `unsupported`

Reply:
You have an incomplete open back tick.  I think the extra back tick isn't needed at all.